src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -595,7 +634,8 @@ def load_status_file_components(self, initial_load=False):
                 if self.execution_config.exec_auto_assess_only:
                     self.__installation_substatus_json = complete_status_file_data['status']['substatus'][i]
                 else:
-                    self.__installation_summary_json = self.__get_substatus_message(complete_status_file_data, i)
+                    self.__installation_summary_json = self.__get_substatus_msg_at_index(complete_status_file_data, i)
```

Comment:
nit: function rename is not needed, the original function name also is reflective of fetching a singular substatus from the status file. 
NOTE: Try avoiding changes unrelated to the truncation logic, as much as possible, in this PR since it is pretty extensive on it's own

Reply:
the renamed from one of previous comment. change reverted.